package.json

```diff
@@ -194,6 +194,7 @@
     "mockdate": "3.0.5",
     "nock": "13.3.7",
     "nodemon": "3.0.1",
+    "pinyin-pro": "3.17.0",
```

Comment:
Why introduce `pinyin-pro` here, I think format scripts can see this deps from `website/package.json`